src/components/Proposal/View/ProposalPromotionSection.tsx

```diff
@@ -68,19 +70,17 @@ export default function ProposalPromotionSection({ proposal, loading }: Props) {
     }
   }
 
+  const isPromoteDisabled =
+    (type === ProposalType.Pitch && (hasTenderProcessFinished || hasTenderProcessStarted)) ||
```

Comment:
If `hasTenderProcessFinished` is true, then `hasTenderProcessStarted` should always be true right? 

So this could be 

```typescript
const isPromoteDisabled =
    (type === ProposalType.Pitch && hasTenderProcessStarted) || type === ProposalType.Tender
```

Unless `hasTenderProcessStarted` means that the tender process is ongoing, and could be false once it's finished, in which case I'd call it `isTenderProcessOngoing` or open.

Reply:
You are absolutely right